Snakemake/workflow/rules/phasing.smk

```diff
@@ -19,18 +19,21 @@ else:
         params: 
             map = f'{mapdir}/{{chromosome}}.txt',
         #log: 'logs/phase_{chromosome}.log'
-        threads: 1
+        threads: 25
```

Comment:
I think this number should be put in the config

Reply:
I'm just using as a variable.